integrations/slack.py

```diff
@@ -187,88 +180,143 @@ def generate_msg(alert: any, options: any) -> any:
+
+    Raises
+    ------
+    FileNotFoundError
+        If no JSON file is found.
+    JSONDecodeError
+        If no valid JSON file is used.
     """
     try:
-        with open(file_location) as alert_file:
+        with open(file_location, encoding='utf-8') as alert_file:
             return json.load(alert_file)
     except FileNotFoundError:
-        debug("# JSON file for alert %s doesn't exist" % file_location)
+        logger.error("JSON file for alert %s doesn't exist", file_location)
         sys.exit(ERR_FILE_NOT_FOUND)
     except json.decoder.JSONDecodeError as e:
-        debug("Failed getting JSON alert. Error: %s" % e)
+        logger.error("Failed getting JSON alert. Error: %s", e)
         sys.exit(ERR_INVALID_JSON)
 
 def get_json_options(file_location: str) -> any:
-    """
-        Read JSON options object from file
-
-        Parameters
-        ----------
-        file_location : str
-            Path to the JSON file location.
-
-        Returns
-        -------
-        {}: any
-            The JSON object read it.
-
-        Raises
-        ------
-        JSONDecodeError
-            If no valid JSON file are used
+    """Read JSON options object from file
+
+    Parameters
+    ----------
+    file_location : str
+        Path to the JSON file location.
+
+    Returns
+    -------
+    any
+        The JSON object read it.
+
+    Raises
+    ------
+    JSONDecodeError
+        If no valid JSON file is used
     """
     try:
-        with open(file_location) as options_file:
+        with open(file_location, encoding='utf-8') as options_file:
             return json.load(options_file)
     except FileNotFoundError:
-        debug("# JSON file for options %s doesn't exist" % file_location)
+        logger.error("JSON file for options %s doesn't exist", file_location)
     except BaseException as e:
-        debug("Failed getting JSON options. Error: %s" % e)
+        logger.error("Failed getting JSON options. Error: %s", e)
         sys.exit(ERR_INVALID_JSON)
 
+def print_help_msg():
+    help_msg = '''Exiting: Invalid arguments.
+    
+Usage:
+    slack <alerts_file> [api_key] <webhook_url> [logging_level] [options_file]
+Arguments:
+    alerts_file (required)
+        Path to the JSON file containing the alerts.
+    api_key (not required)
+        The API key argument is not needed for the Slack integration. However, it's still considered because the 
+        integrator executes all scripts with the same arguments.
+        If you are executing the script manually, please put anything in that argument.
+    webhook_url (required)
+        Slack webhook URL where the messages will be sent to.
+    logging_level (optional)
+        Used to define how much information should be logged. Default is INFO.
+        Levels: NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL.
+    options_file (optional)
+        Path to a file containing custom variables to be used in the integration. It must be JSON-encoded.
+    '''
```

Comment:
```suggestion
    help_msg = '''
    Exiting: Invalid arguments.
    
    Usage:
        slack <alerts_file> [api_key] <webhook_url> [logging_level] [options_file]
    Arguments:
        alerts_file (required)
            Path to the JSON file containing the alerts.
        api_key (not required)
            The API key argument is not needed for the Slack integration. However, it's still considered because the 
            integrator executes all scripts with the same arguments.
            If you are executing the script manually, please put anything in that argument.
        webhook_url (required)
            Slack webhook URL where the messages will be sent to.
        logging_level (optional)
            Used to define how much information should be logged. Default is INFO.
            Levels: NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL.
        options_file (optional)
            Path to a file containing custom variables to be used in the integration. It must be JSON-encoded.
    '''
```